x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -13,6 +13,7 @@
 import org.elasticsearch.common.logging.HeaderWarning;
 import org.elasticsearch.common.logging.LoggerMessageFormat;
 import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.enrollment.EnrollmentSettings;
```

Comment:
Why do we want to tie enrollement mode with the License State in any way ? The value and the availability of the setting is not dependent on the license

Reply:
This is still pending